.github/workflows/bump-bulk-cdk-and-release-connectors.yml

```diff
@@ -10,8 +10,93 @@ on:
         options:
           - airbytehq/airbyte
 jobs:
-  bump-cdk-version-and-merge:
+  publish-bulk-cdk:
+    uses: ./.github/workflows/publish-bulk-cdk.yml
+    secrets: inherit
+  bump-cdk-version:
     runs-on: ubuntu-24.04
+    outputs:
+      # Output the matrix config as a JSON string
+      modified_connectors: ${{ steps.export-connection-modified.outputs.modified_connectors }}
+    needs: [publish-bulk-cdk]
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+        with:
+          # Fetch all history for all tags and branches
+          fetch-depth: 0
+      - name: Fetch version from repo
+        id: fetch-version
+        run: |
+          # 1. Define the target URL
+          REPO_URL="https://airbyte.mycloudrepo.io/public/repositories/airbyte-public-jars/io/airbyte/bulk-cdk/bulk-cdk-core-load/"
+          echo "Fetching versions from $REPO_URL" >&2 # Log output to standard error
+
+          # 2. Fetch HTML, extract versions (X.Y), sort, get the latest
+          latest_version=$(curl -H 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8' -s "$REPO_URL" | grep -o 'href="[0-9]\+\.[0-9]\+/"' | sed 's/^href="//; s/\/"$//' | sort -V | tail -n 1)
+          echo "Latest version found: $latest_version"
+
+          # 3. Set the output parameter for GitHub Actions
+          # This makes "latest_version" available to subsequent steps
+          echo "latest_version=$latest_version" >> "$GITHUB_OUTPUT"
+      - name: Configure Git
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Update cdk in build.gradle files
+        id: export-connection-modified
+        run: |
+          ./tools/bin/cdk-auto-release/bump_cdk_version.sh ${{ steps.fetch-version.outputs.latest_version }}
+
+      - name: Commit changes
+        run: |
+          git add .
+          git commit -m "Bump the cdk version to ${{ steps.fetch-version.outputs.latest_version }}"
+          git push
+
+  publish-connectors:
+    needs: [bump-cdk-version] #, publish-bulk-cdk]
```

Comment:
```suggestion
    needs: [bump-cdk-version] 
```

Reply:
It shouldn't have been commented but needing `publish-bulk-cdk` is required because the matrix is using the output of `publish-bulk-cdk`.